cockatrice/src/carddatabase.cpp

```diff
@@ -291,22 +292,22 @@ void CardInfo::refreshCachedSetNames()
 
 QString CardInfo::simplifyName(const QString &name)
 {
-    QString simpleName(name);
+    static const QRegularExpression nonAlnumspace("[^a-z0-9 ]");
```

Comment:
can we come up with a better variable name

Reply:
it matches anything that is not alphanumeric or a space